src/components/edit-asset-ownership-form/edit-asset-ownership-form.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from "react";
+import { Link as RouterLink } from "react-router-dom";
+
+import locale from "../../services/locale";
+
+import { Center, Spinner } from "@mtfh/common";
+import {
+  Asset,
+  PatchAssetLbhOwnershipRequest,
+  patchAsset,
+} from "@mtfh/common/lib/api/asset/v1";
+import "./styles.scss";
+
+interface EditAssetOwnershipFormProps {
+  setShowSuccess: React.Dispatch<React.SetStateAction<boolean>>;
+  setRequestError: React.Dispatch<React.SetStateAction<string | null>>;
+  asset: Asset;
+}
+
+export const EditAssetOwnershipForm = ({
+  setShowSuccess,
+  setRequestError,
+  asset,
+}: EditAssetOwnershipFormProps) => {
+  const [isCouncilProperty, setIsCouncilProperty] = useState<boolean>(
+    asset.assetManagement.isCouncilProperty,
+  );
+  const [formSubmitted, setFormSubmitted] = useState<boolean>(false);
+  const [loading, setLoading] = useState<boolean>(false);
+
+  const handleRadioButton = (value: boolean) => {
+    setIsCouncilProperty(value);
+  };
+
+  const handleFormSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setLoading(true);
+
+    const { assetManagement } = asset;
+
+    const requestPayload: PatchAssetLbhOwnershipRequest = {
+      assetManagement: {
+        agent: assetManagement?.agent,
+        areaOfficeName: assetManagement?.areaOfficeName,
+        isCouncilProperty,
+        managingOrganisation: assetManagement?.managingOrganisation,
+        managingOrganisationId: assetManagement?.managingOrganisationId,
+        owner: assetManagement?.owner,
+        isTMOManaged: assetManagement?.isTMOManaged,
+        propertyOccupiedStatus: assetManagement?.propertyOccupiedStatus,
+        propertyOccupiedStatusReason: assetManagement?.propertyOccupiedStatusReason,
+        isNoRepairsMaintenance: assetManagement?.isNoRepairsMaintenance,
+        councilTaxType: assetManagement?.councilTaxType,
+        councilTaxLiability: assetManagement?.councilTaxLiability,
+        isTemporaryAccomodation: assetManagement?.isTemporaryAccomodation,
+        readyToLetDate: assetManagement?.readyToLetDate,
+      },
```

Comment:
The parameter names mostly match. You could instead so something like:

```js
{
    assetManagement: {
        ...assetManagement,
        isCouncilProperty
    }

}
```

Reply:
Changed, nice suggestion